File: Práctica 1/Java/Otras pruebas/test/zCol03.java
import java.util.ArrayList;

public class zCol03 {
	public static void imprimirMapa(ArrayList<ArrayList<Character>> mapa){
		for(int i = 0; i < mapa.size(); i++) {
			for(int j = 0; j < mapa.get(i).size(); j++) {
				System.out.print(mapa.get(i).get(j));
			}
			System.out.println();
		}
	}
	public static void imprimirLocalidades(ArrayList<Localidad> localidades) {
		for(int i = 0; i< localidades.size(); i++) {
			System.out.println(localidades.get(i).toString());
		}
	}
	public static void main(String [] args) {
		Coleccion col = new Coleccion();
		ArrayList<Localidad> locas;
		ArrayList<ArrayList<Character>> mapa;
		
		col.lectura("zCol03.dat");
		mapa = col.getMapa();
		locas = col.getLocalidades();
		imprimirMapa(mapa);
		imprimirLocalidades(locas);
	
		for(int i = -1; i <= (int) mapa.size(); i++) {
			for(int j = -1; j <= (int) mapa.get(0).size(); j++) {
				System.out.print(col.getCoorMapa(new Coordenadas(i, j)));
			}
			System.out.println();
		}
	}
}

File: Práctica 1/C++/practica1-prueba/zCol03.cc
#include <iostream>
using namespace std;

#include "Localidad.h"
#include "Coordenadas.h"
#include "Coleccion.h"
#include "InfoTur.h"


void imprimirMapa(vector< vector<char> > mapa){
	for(int i = 0; i < mapa.size(); i++){
		for(int j = 0; j < mapa[i].size(); j++){
			cout << mapa[i][j];
		}
		cout << endl;
	}
}
void imprimirLocalidades(vector< Localidad > localidades){
	for(int i = 0; i < localidades.size(); i++){
		cout << localidades[i] << endl;
	}
}


int main(){
	Coleccion col;
	vector<Localidad> locas;
	vector< vector<char> > mapa;

	col.lectura("zCol03.dat");
	mapa = col.getMapa();
	locas = col.getLocalidades();

	imprimirMapa(mapa);
	imprimirLocalidades(locas);

	char elemento;
	
	for(int i = -1; i <= (int) mapa.size(); i++){
		for(int j = -1; j <= (int) mapa[0].size(); j++){
			Coordenadas c(i, j);
			cout << col.getCoorMapa(c);
		}
		cout << endl;
	}

	return 0;
}
